src/utils.py

```diff
@@ -99,3 +110,37 @@ def print_os_environment():
     """
     for key, val in os.environ.items():
         print(f"{key}: {val}")
+
+
+def get_semver_command(body):
```

Comment:
the `command` name is confusing, can we call it label or user_imput or something else?